docs/rfc/active/azkaban-flow-job/README.md

```diff
@@ -0,0 +1,95 @@
+- Start Date: 2020-08-25
+- RFC PR: https://github.com/linkedin/datahub/pull/1820
+- Implementation PR(s): https://github.com/linkedin/datahub/pull/1732
+
+# Azkaban Flows and Jobs
+
+## Summary
+
+Adding support for [Azkaban](https://azkaban.github.io/) job and flow metadata and enabling search and discovery for them. 
+
+The design includes the metadata needed to represent Azkaban jobs and flows as data job entities and their relationships to other
+entities like Datasets.
+
+## Motivation
+
+Azkaban is a popular open source workflow manager created and extensively used at LinkedIn. Azkaban metadata is a critical piece
+in the metadata graph since data processing jobs are the primary driver of data movement and creation.
+
+Without job metadata, it is not possible to understand the data flow across an organization. Additionally, jobs are needed in the
+lineage graph to surface operational metadata and have a complete view of data movement and processing
+
```

Comment:
You might want to extend this to capture the DAG of jobs and flows ?

Reply:
Added